src/hooks/useCheckMobileScreen.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+
+export const useCheckMobileScreen = (): boolean => {
```

Comment:
`CheckMobileScreen` wskazuje że to co zwróci hook to funkcja. Lepszą nazwą będzie `useIsMobileScreen`